lib/capistrano/tasks/delayed_job.rake

```diff
@@ -15,6 +15,9 @@ namespace :delayed_job do
     unless fetch(:delayed_job_pools).nil?
       args << fetch(:delayed_job_pools, {}).map {|k,v| "--pool='#{k}:#{v}'" }.join(' ')
     end
+    unless fetch(:delayed_job_daemon_opts).nil?
+      args << fetch(:delayed_job_daemon_opts, []).map { |option| "--daemon-options='--#{option}'"}.join(' ')
```

Comment:
given the documentation change, 
```
set :delayed_job_daemon_opts, ["no_wait", "shush"]
```
are you expecting to add:

```
--daemon-options='--no_wait' --daemon-options='--shush'
```

Or were you wanting to produce:

```
--daemon-options='--no_wait --shush'
```